setup.cfg

```diff
@@ -78,6 +81,12 @@ omit =
     *bin/pyproject-build
     *bin\pyproject-build.exe
 
+[coverage:report]
+exclude_lines =
+    \#\s*pragma: no cover
```

Comment:
Why?

Reply:
Here, the core goal is to add ``raise NotImplementedError`` to be automatically marked with no coverage needed for them (because the metaclass makes it impossible to hit it). By default, coverage only excludes `pragma no cover` lines, so we want to extend that. There's no extend the exclude method, though, override of the exclude lines matcher, so we need to set it again.